pyproject.toml

```diff
@@ -35,10 +35,12 @@ profile = "black"
 
 [tool.pyright]
 include = ["playwright", "tests", "scripts"]
+exclude = ["**/node_modules", "**/__pycache__", "**/.*", "./build"]
 pythonVersion = "3.8"
 reportMissingImports = false
 reportTypedDictNotRequiredAccess = false
 reportCallInDefaultInitializer = true
 reportOptionalSubscript = false
 reportUnboundVariable = false
 strictParameterNoneValue = false
+reportIncompatibleVariableOverride = false

```

Comment:
What is this one? Do we actually want to enable it?

Reply:
We override `self._loop` from being `Optional` to being not `Optional`. Mypy doesn't allow it anymore after they enabled this setting. Keeping it disabled seems fine for now imo.